lib/eclipse/EclipseState/Schedule/WellProductionProperties.cpp

```diff
@@ -86,6 +86,9 @@ namespace Opm {
                 throw std::invalid_argument("Setting CMODE to unspecified control");
         }
 
+        p.BHPH = record.getItem("BHP").getSIDouble(0);
```

Comment:
The `BHP` and `THP` items do not have default values, so this must be "protected" with `hasValue(0)` checks.